apps/etterlatte-behandling/src/test/kotlin/grunnlagsendring/doedshendelse/kontrollpunkt/DoedshendelseKontrollpunktServiceTest.kt

```diff
@@ -27,41 +31,74 @@ class DoedshendelseKontrollpunktServiceTest {
     private val kontrollpunktService = DoedshendelseKontrollpunktService(pdlTjenesterKlient, pesysKlient)
     private val doedshendelse =
         Doedshendelse.nyHendelse(
-            avdoedFnr = "12345678901",
+            avdoedFnr = KONTANT_FOT.value,
             avdoedDoedsdato = LocalDate.now(),
-            beroertFnr = "109987654321",
+            beroertFnr = JOVIAL_LAMA.value,
             relasjon = Relasjon.BARN,
         )
 
-    @Test
-    fun `Skal returnere kontrollpunkt hvis avdoed ikke har doedsdato i PDL`() {
+    @BeforeEach
+    fun oppsett() {
         coEvery { pesysKlient.hentSaker(doedshendelse.beroertFnr) } returns emptyList()
+
         every {
             pdlTjenesterKlient.hentPdlModell(
                 foedselsnummer = doedshendelse.avdoedFnr,
                 rolle = PersonRolle.AVDOED,
                 saktype = any(),
             )
-        } returns mockPerson()
-
-        val kontrollpunkter = kontrollpunktService.identifiserKontrollerpunkter(doedshendelse)
-
-        kontrollpunkter shouldContainExactly listOf(DoedshendelseKontrollpunkt.AvdoedLeverIPDL)
+        } returns
```

Comment:
dette trengs å gjøres for hver metofde?
ha en metode som mekker dette oppsettet? 🐮

Reply:
Det ligger jo allerede i en metode som mekker dette oppsettet? Som heter `oppsett` 😁.
PDL blir kalt for alle disse tre personene, og det forventes at det ser ut på en viss måte for at vi skal slippe de igjennom. Testene under overstyrer noen av disse verdiene.